Desktop/vsgviewer/vsgviewer.cpp

```diff
@@ -59,7 +59,7 @@ int main(int argc, char** argv)
     // read any vsg files
     for (int i=1; i<argc; ++i)
     {
-        vsg::Path filename = arguments[i];
+        vsg::Path filename = searchPaths[0]+arguments[i];
```

Comment:
searchPaths could easily be empty so would crash on this line.

If we want to use use the searchPaths then use the vsg::findFile() function. i.e.

     auto filename = vsg::findFile(arguments[i], searchPaths);

Reply:
I've just looked up vsg::findFIle() implementation and it return {} when the file doesn't exist in the specified paths, and if the searchPaths doesn't include an empty path entry it won't test if the file is in the current directory.

It may be worth me adding an option to check the filename as is without any path, or add a convenience function that searches the searchPaths and checks the original filepath as well.  

Without this one would probably want to do something like:

  vsg::Path filename = vsg::fileExists(arguments[i]) ? arguments[i] : vsg::fileFile(arguments[i], searchPaths);

Clunky though...  I'll ponder on this issue.  For this PR just remove the above change.

Reply:
it's hack artefact will remove lmy changes in vsgviewer